src/chainer.js

```diff
@@ -1,8 +1,11 @@
 /**
+ *
  * @param {function[]} functions
  *
  * @returns {function}
  */
+'use strict';
```

Comment:
The 'use strict' statement is unnecessary in ES6 modules because ES6 is always strict mode by default.